README.md

```diff
@@ -127,28 +155,28 @@ curl -X 'POST' \
   | jq -C . | less -R
 ```
 
-#### Coordinates
+### Coordinates
 
 When elements are extracted from PDFs or images, it may be useful to get their bounding boxes as well. Set the `coordinates` parameter to `true` to add this field to the elements in the response.
 
 ```
  curl -X 'POST' \
-  'https://api.unstructured.io/general/v0/general' \
+  'https://api.unstructuredapp.io/general/v0/general' \
   -H 'accept: application/json' \
   -H 'Content-Type: multipart/form-data' \
   -F 'files=@sample-docs/layout-parser-paper.pdf' \
   -F 'coordinates=true' \
   | jq -C . | less -R
 ```
 
-#### Skip Table Extraction
+### Skip Table Extraction
```

Comment:
Consider using "Sentence case" for headings instead of "Title Case," for consistency. For example, in this case, "Skip table extraction."